unit-tests/test-phase.js

```diff
@@ -224,4 +225,23 @@ describe('phase', () => {
       assert.isFunction(phase.triggerGC);
     });
   });
+
+  describe('timeoutError', () => {
+
+    it('error thrown', () => {
+
+      let errorFunc = () => {
+        phase.timeoutError();
+      }
+
+      assert.throw(errorFunc, 'Test timeout exceeded ' + OPTIONS.timeout + 'ms');
```

Comment:
Yeah it has to be in it's own function. If I have:

Then it throws the error immediately and chai doesn't catch it:

assert.throw(phase.timeoutError(), 'Test timeout exceeded ' + OPTIONS.timeout + 'ms');
...

  1 failing

  1) phase timeoutError error thrown:
     Error: Test timeout exceeded 30000ms
      at Phase.timeoutError (lib/phases/phase.js:83:5)
      at Context.<anonymous> (unit-tests/test-phase.js:235:26)

npm ERR! Test failed.  See above for more details.